### PR TITLE
nc-news-5 complete with endpoints.json updated

### PR DESCRIPTION
**Pull request by C43NK0**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -103,3 +103,29 @@ describe("Get /api/articles/:article_id", () => {
...
 
 })
+
+describe("Get /api/articles", () => {
+    test("200: responds with an articles array of article objects with the relevant properties, sorted by date in descending order and without a body property", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.articles).toBeInstanceOf(Array)
```
Comment (RichardNeat) [inline __tests__/app.test.js:114]: Nice idea with the below, but we could have used a forEach here to check that each element looks as expected to have better coverage.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -103,3 +103,29 @@ describe("Get /api/articles/:article_id", () => {
...
+            article_id: 6,
+            topic: 'mitch',
+            created_at: '2020-10-18T01:00:00.000Z',
+            votes: 0,
+            article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+            comment_count: '1'
+            })
+        })
+    })
+})
```
Comment (RichardNeat) [inline __tests__/app.test.js:142]: Could we also check for:
- A test that proves the body doesn't come back
- The results are ordered by date
The above two points are the extra tasks on learn to code towards the bottom

**Review by RichardNeat — APPROVED** [review]
Nice one Alex - just a couple of extra tests to consider in the comments below. Could you make those changes please then resubmit :)

File: __tests__/app.test.js [hunk 3]
```diff
@@ -112,7 +113,17 @@ describe("Get /api/articles", () => {
...
+            
+            body.articles.forEach((item) => {
+                expect(item).toHaveProperty("comment_count")})
+
+                for (let i = 0; i < body.articles.length -1; i++){
+                    let latest = new Date(body.articles[i].created_at);
+                    let next = new Date(body.articles[i + 1].created_at);
+                    console.log(latest, next)
+                    expect(next.getTime()).toBeLessThanOrEqual(latest.getTime())
+                }
```
Comment (Dutton1990) [inline __tests__/app.test.js:125]: Nice idea here! Next time have a look at jest-sorted :)

File: __tests__/app.test.js [hunk 4]
```diff
@@ -112,7 +113,17 @@ describe("Get /api/articles", () => {
         .then(({ body }) => {
             expect(body.articles).toBeInstanceOf(Array)
             expect(body.articles).toHaveLength(13)
-            expect(body.articles[10]).toHaveProperty("comment_count")
+            
+            body.articles.forEach((item) => {
+                expect(item).toHaveProperty("comment_count")})
```
Comment (Dutton1990) [inline __tests__/app.test.js:118]: Nice refactor!

**Review by Dutton1990 — APPROVED** [review]
Nice job Alex you can merge this one!

**Pull request merged**